marimo/_server/api/endpoints/assets.py

```diff
@@ -31,9 +32,14 @@
 # Root directory for static assets
 root = os.path.realpath(str(import_files("marimo").joinpath("_static")))
 
+config = UserConfigManager().get_config().get("server", {})
```

Comment:
I wasn't sure how to get the MarimoConfig from a top-level context like this. Most call sites seem to read it off an existing object. Is there a better way than re-instantiating a UserConfigManager?